### ISSUE TITLE
Deploy is failed.

### ISSUE THREAD

**Issue by gkaxowns9** [ISLAND-IDs/Website#4]
### <span aria-hidden="true">❌</span> Deploy Preview for *island-promotion* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 992b1ea42b2736b0ae0159d2adf9ed89784d28a3 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/island-promotion/deploys/68d22abf931e450008e164ec |

_Originally posted by @netlify[bot] in https://github.com/ISLAND-IDs/Website/issues/3#issuecomment-3322427103_
